packages/@react-aria/toast/src/useToast.ts

```diff
@@ -10,17 +10,16 @@
  * governing permissions and limitations under the License.
  */
 
+import {chain} from '@react-aria/utils';
 import {DOMProps} from '@react-types/shared';
 import {HTMLAttributes, ImgHTMLAttributes} from 'react';
 import intlMessages from '../intl/*.json';
 import {PressProps} from '@react-aria/interactions';
-import {ToastProps} from '@react-types/toast';
+import {SpectrumToastProps} from '@react-types/toast';
 import {useId} from '@react-aria/utils';
 import {useMessageFormatter} from '@react-aria/i18n';
 
-interface AriaToastProps extends ToastProps {
-  id?: string
-}
+interface AriaToastProps extends SpectrumToastProps {}
```

Comment:
I think `AriaToastProps` shouldn't extend `SpectrumToastProps` since this is an aria hook and we want to keep any spectrum specific props out of the prop definition as possible. Perhaps `ToastProps` should be expanded to extend ToastState and any other props that this hook needs?

Reply:
I'm looking ad the code and I don't recall why I called this `SpectrumToastProps`, I might have meant `ReactToastPRops` it should probably be renamed. However, sure, I have AriaToastProps extends ToastProps and ToastState with a `toastKey`. I think I extended `SpectrumToastProps` because I was trying to define all the variables like `toastKey` in one place and not spread around the code. I can probably accomplish that a different way.

Reply:
I think `SpectrumToastProps` is a more appropriate name than `ReactToastProps`
I think it should be `ToastAriaProps` instead of `AriaToastProps`

Reply:
I went looking at other aria components and it seems inconsistent. I see `AriaToastProps`, `ToastAriaProps` and `ToastProps` for the props passed into an aria hook.